docs/source/markdown/podman-mount.1.md.in

```diff
@@ -41,31 +41,25 @@ Do not truncate the output (default *false*).
 
 ## EXAMPLE
 
+Mount specified container. If you do this in rootless mode, make sure you do podman unshare first:
```

Comment:
As for image- and volume-mount, I'd love one example for rootless and another for rootful.

Reply:
Lets get one merged then I will retrofit the others.  podman-volume-mount is documenting both I believe.